src/netex-convertor/single-ticket/singleTicketNetexGenerator.ts

```diff
@@ -10,10 +15,15 @@ import {
 } from './singleTicketNetexHelpers';
 import { NetexObject, getCleanWebsite, getNetexTemplateAsJson, convertJsonToXml } from '../sharedHelpers';
 
```

Comment:
Do we not need a check for `(ticket as MatchingReturnData).outboundFareZones !== undefined`? With `const isReturnTicket = (ticket: MatchingData): ticket is MatchingReturnData => ((ticket as MatchingReturnData).inboundFareZones !== undefined && (ticket as MatchingReturnData).inboundFareZones.length > 0) ||
    ((ticket as MatchingReturnData).outboundFareZones !== undefined && (ticket as MatchingReturnData).outboundFareZones.length > 0);` the `make-generate-point-to-point` script in fdbt-dev no longer fails